src/sentry/models/groupassignee.py

```diff
@@ -131,6 +164,9 @@ def assign(
             ):
                 sync_group_assignee_outbound(group, assigned_to.id, assign=True)
 
+        if not created:  # aka re-assignment
```

Comment:
It looks like there is already a `if not created` block above. Can this move up there or does it need to happen in this order?

Reply:
Yeah it needs to be lower otherwise the notifications won't send.